services/src/test/java/org/keycloak/vault/KeystoreVaultProviderTest.java

```diff
@@ -0,0 +1,74 @@
+package org.keycloak.vault;
+
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.Test;
+import org.keycloak.common.util.Environment;
+
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.keycloak.vault.SecretContains.secretContains;
+
+/**
+ * Tests for {@link FilesKeystoreVaultProvider}.
+ *
+ * @author Peter Zaoral
+ */
+public class KeystoreVaultProviderTest {
+
+    @Before
+    public void before() {
+        // TODO: improve when the supported keystore types for FIPS will be unified across the codebase
+        Assume.assumeFalse("Java is in FIPS mode. Skipping the test.", Environment.isJavaInFipsMode());
```

Comment:
I think this `assume` is not quite correct. Do we really want to run this test just in FIPS mode?

Reply:
Sorry, it's the other way around. I got confused :)